activesupport/test/digest_test.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "abstract_unit"
+require "openssl"
+
+class DigestTest < ActiveSupport::TestCase
+  def test_with_md5_hash
+    assert_equal ::Digest::MD5.hexdigest("hello friend"), ActiveSupport::Digest.hexdigest("hello friend")
```

Comment:
I think you can use md5 hashed value instead of calling MD5 here in specs:

`assert_equal "1079a055f110d54ba12f08bd6b671f6c", ActiveSupport::Digest.hexdigest("hello friend")`